### ISSUE TITLE
Are test details required when the test fails overall?

### ISSUE THREAD

**Issue by guygastineau** [exercism/docs#355]
https://github.com/exercism/docs/blob/a614e2a1432b346df7e7bb5bd415c7f36a626e81/building/tooling/test-runners/interface.md?plain=1#L66

I didn't find an example with a root status of `error`, and I am wondering if the `tests` field is required for this case.  I am retrofitting the test system and runner for the scheme track to be v2 or 3 conformant.  Currently I am passing the "code" for each test as a result of the test system, but that information is not sent when there is compiler/interpreter error.  For now I just have tests' value set to null after including a message for error statuses, but I want to make sure I am following the spec.

TL;DR;
Do I need to include the tests field when the status is `error` and a `message` is provided?  If not, should it be not present, `"tests": null"`, or either of these?  Maybe it would be an empty array, `"tests": []`?

**Comment by ErikSchierboom** [comment]
Good question. I've created [a PR](https://github.com/exercism/docs/pull/356) to clarify.

**Comment by guygastineau** [comment]
Thank you!
